spec/requests/users_spec.rb

```diff
@@ -1,28 +1,28 @@
-require 'rails_helper'
+# require 'rails_helper'
 
-RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
-    it 'should returns http 200 success' do
-      get users_path
-      expect(response).to have_http_status(200)
-    end
+# RSpec.describe 'Users', type: :request do
+#   describe 'GET /index' do
+#     it 'should returns http 200 success' do
+#       get users_path
+#       expect(response).to have_http_status(200)
+#     end
 
-    it 'should include Number of posts' do
-      get users_path
-      expect(response.body).to include('Number of posts')
-    end
-  end
+#     it 'should include Number of posts' do
+#       get users_path
+#       expect(response.body).to include('Number of posts')
+#     end
+#   end
 
-  describe 'GET /show' do
-    user = User.create('name' => 'Amy', 'bio' => 'bio', 'photo' => 'Tom.png', 'posts_counter' => 0)
-    before(:each) { get user_path id: user.id }
+#   describe 'GET /show' do
+#     user = User.create('name' => 'Amy', 'bio' => 'bio', 'photo' => 'Tom.png', 'posts_counter' => 0)
+#     before(:each) { get user_path id: user.id }
 
-    it 'should return correct response' do
-      expect(response).to have_http_status(200)
-    end
```

Comment:
- [ ] I would recommend removing all the commented code in different files if you are no longer using them.

Reply:
Thank you for your feedback. I'll fix it.